src/UXClient/Components/Marker/Marker.ts

```diff
@@ -72,11 +72,17 @@ class Marker extends Component {
         text.append('h4')
             .attr('class', 'tsi-seriesLabelGroupName tsi-tooltipTitle')
             .text(d.aggregateName);
+        
+        let shiftTuple = this.chartComponentData.getTemporalShiftStringTuple(d.aggregateKey);
+        let shiftString = '';
+        if (shiftTuple !== null) {
+            shiftString = shiftTuple[0] === 'Start at' ? this.timeFormat(new Date(shiftTuple[1])) : shiftTuple[1]; 
```

Comment:
Can 'Start at' be made into a string constant since you're using it to test for equality and it's used in multiple places (like ChartComponent.ts and ChartComponentData.ts)?

Reply:
yes, definitely should be, good point!